docs/reference/tmx-map-format.rst

```diff
@@ -249,7 +263,9 @@ number of Wang tiles using these colors.
 -  **name**: The name of the Wang set.
 -  **tile**: The tile ID of the tile representing this Wang set.
 
-Can contain: :ref:`tmx-wangcornercolor`, :ref:`tmx-wangedgecolor`, :ref:`tmx-wangtile`
+Can contain at most one: :ref:`tmx-wangcornercolor`, :ref:`tmx-wangedgecolor`
```

Comment:
Actually can contain up to 15 of each.

Reply:
Did not know that. The number 15 never actually appears in the docs. Added!